glue/VerboseManagerImpl.cpp

```diff
@@ -85,7 +85,8 @@ MM_VerboseManagerImpl::reconfigureVerboseGC(OMR_VM *omrVM)
 	 * otherwise we append the pid of the child before the extension.
 	 */
 	WriterType type = parseWriterType(NULL, filename, 0, 0); /* All parameters other than filename aren't used */
-	if (type == VERBOSE_WRITER_FILE_LOGGING && NULL == strstr(filename, "%p") &&  NULL == strstr(filename, "%pid")) {
+	if ((type == VERBOSE_WRITER_FILE_LOGGING_SYNCHRONOUS  || type == VERBOSE_WRITER_FILE_LOGGING_BUFFERED)
+			&& NULL == strstr(filename, "%p") &&  NULL == strstr(filename, "%pid")) {
```

Comment:
I know not all of this is new code but can you wrap the 2 NULL == with brackets?  Thanks